cli/src/api/pcluster_api.py

```diff
@@ -65,16 +65,39 @@ def __repr__(self):
         return json.dumps(self.__dict__)
 
 
-class ImageInfo:
+class ImageBuilderInfo:
     """Minimal representation of a building image."""
 
     def __init__(self, imagebuilder: ImageBuilder):
-        self.image_name = imagebuilder.stack.name
-        self.imagebuild_status = imagebuilder.imagebuild_status
-        self.stack_status = imagebuilder.stack.status
-        self.stack_arn = imagebuilder.stack.id
+        self.stack_exist = False
+        self.image_exist = False
         self.region = get_region()
-        self.version = imagebuilder.stack.version
+        # image config file url
+        self.image_configurtion = imagebuilder.config_url
+
+        if AWSApi.instance().cfn.stack_exists(imagebuilder.image_name):
```

Comment:
Why do you have the same changes in the Pcluster delete/describe patch? You might unify the two patches.

Reply:
will rebase this PR after delete-image and describe-image commands get merged.